azure-iot-device/tests/iothub/aio/test_async_handler_manager.py

```diff
@@ -229,30 +229,38 @@ async def test_handler_invoked_multiple(
     async def test_handler_resolve_pending_items_before_handler_removal(
         self, mocker, handler_name, handler_manager, handler, handler_checker, inbox
     ):
-        # Set the handler
-        setattr(handler_manager, handler_name, handler)
-        # Add 100 items to the associated inbox (enough to build up a backlog)
+        assert inbox.empty()
```

Comment:
This change here is unrelated to the rest of the PR - it's a fix for a flaky test that was giving me trouble upon trying to check in